--- a/layers/meta-qt5/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb	2018-03-21 14:00:07.646757697 +0800
+++ b/layers/meta-qt5/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb	2018-03-21 14:05:51.554750109 +0800
@@ -104,6 +104,9 @@
     qtwebchannel-dev \
     qtwebchannel-mkspecs \
     ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'qtwebchannel-qmlplugins', '', d)} \
+    qtwebengine-dev \
+    qtwebengine-mkspecs \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'qtwebengine-qmlplugins', '', d)} \
     qtxmlpatterns-dev \
     qtxmlpatterns-mkspecs \
     qttranslations-qtxmlpatterns \
